sympy/core/expr.py

```diff
@@ -3584,6 +3584,10 @@ def leadterm(self, x, logx=None, cdir=0):
         c = c.subs(d, log(x))
         return c, e
 
+    @overload
+    def as_coeff_Mul(self, rational: Literal[True]) -> tuple['Rational', Expr]: ...
+    @overload
+    def as_coeff_Mul(self, rational: bool = False) -> tuple['Number', Expr]: ...
     def as_coeff_Mul(self, rational: bool = False) -> tuple['Number', Expr]:
```

Comment:
I think it is better to have a blank line between the the two overloads and then the actual function signature.

Reply:
This applies to the other places as well.